docs/getting_started.md

```diff
@@ -10,211 +10,179 @@ We use Azure DevOps for running our build(CI), retraining trigger and release (C
-    - Click Add
-    - Click on lighting sign to enable the **Continous Deployment Trigger**, click **Save**.
-    ![release retraining artifact](./images/model-artifact-cd-trigger.png)
+![trained model](./images/trained-model.png)
+
+Good! Now we have a trained model.
+
+### 8. Deploy the Model
+
+The final step is to deploy the model across environments with a release pipeline. There will be a **``QA``** environment running on [Azure Container Instances](https://azure.microsoft.com/en-us/services/container-instances/) and a **``Prod``** environment running on [Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service). 
+
+![deploy model](./images/deploy-model.png)
 
 
-1. We now have QA environment continously deployed each time there is a new ml model registered in AML Model Management. You can select pre-deployment conditions for prod environment, normally you don't want it to be auto deployed, so select manual only trigger here.
+This pipeline leverages the **Azure Machine Learning** extension that should be installed in your organization from the [marketplace](https://marketplace.visualstudio.com/items?itemName=ms-air-aiagility.vss-services-azureml).
 
-    ![release retraining artifact](./images/release-deploymentprodtrigger.png)
+The pipeline consumes two artifacts: the result of the **Build Pipeline** as it contains configuration files and the **model** trained and registered by the ML training pipeline. 
 
-    To deploy a release manually, follow the document [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-designer?view=azure-devops&tabs=new-nav#deploy-a-release)
+Configuration of a code **_ci-build** artifact is similar to what we did in the previous chapter. 
 
+In order to configure a model artifact there should be a service connection to **mlops-AML-WS** workspace:
 
-Congratulations, you now have three pipelines set up end to end.
-   - Build pipeline: triggered on code change to master branch on GitHub.
-   - Release Trigger pipeline: triggered on build pipeline execution and registers a new ML model to AML Model Management if better than previous one.
-   - Release Deployment pipeline: QA environment is auto triggered when there is a new model.
-    Prod is manual only and user decides when to release to this environment.
+![workspace connection](./images/workspace-connection.png)
+
+Add an artifact to the pipeline and select **AzureML Model Artifact** source type. Select the **Service Endpoint** and **Model Names** from the drop down lists:
+
+![model artifact](./images/model-artifact.png)
+
+Create a stage **QA (ACI)** and add a single task to the job **Azure ML Model Deploy**. Specify task parameters as it is shown below:
+
+![deploy aci](./images/deploy-aci.png)
+
+In a similar way create a stage **Prod (AKS** and add a single task to the job **Azure ML Model Deploy**. Specify task parameters as it is shown below:
+

```

Comment:
Please add the `code` snip for the file paths to make the user experience much easier

Reply:
Done